package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/react-connect-js",
-  "version": "3.3.6-beta-1",
+  "version": "3.3.7-beta-1",
   "description": "React components for Connect.js and Connect embedded components",
   "main": "dist/react-connect.js",
   "module": "dist/react-connect.esm.js",
```

Comment:
You need a version publish and bump for connect-js! You can ask for it at #connected-accounts-ui